cmd/adapter-rest/startcmd/testdata/outputdescriptors.json

```diff
@@ -0,0 +1,70 @@
+[
```

Comment:
has to be in `map[string][]OutputDescriptors` format

Reply:
done